script.js

```diff
@@ -0,0 +1,67 @@
+const div = document.querySelector('#bookListContainer');
+const ul = document.querySelector('#ulBookList');
+const addForm = document.getElementById('addBookForm');
+
+class BookCollection {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(cTitle, cAuthor) {
+    const newBook = {
+      title: cTitle,
+      author: cAuthor,
+    };
+    this.books.push(newBook);
+    localStorage.setItem('localStorageBooks', JSON.stringify(this.books));
+  }
+
+  removeBook(index) {
+    const result = this.books.filter((book) => book.title !== this.books[index].title);
```

Comment:
- [ ] **Required:** Currently when I enter a book multiple times with the same title and one is removed, all of the books with that title are removed. Kindly fix this.

Reply:
Hi brytebee.
You are right, we were filtering by the books title. Now we are filtering by the index of the array. 
Thank you very much for your review